src/utils/abi.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * [ERC-20 Token Standard](https://ethereum.org/en/developers/docs/standards/tokens/erc-20)
+ */
+export const erc20ABI = [
```

Comment:
not sure if we need all this, could also do a minimal version, and whether we want this for 721/1155 in the future

Reply:
Think it's okay to have the full ABI for now.